sklearn/ensemble/tests/test_stacking.py

```diff
@@ -0,0 +1,218 @@
+"""
+Testing for the stacking ensemble module (sklearn.ensemble.stacking).
+"""
+
+# Author: Caio Oliveira
+# License BSD 3 clause
+
+
+from copy import deepcopy
+import numpy as np
+from scipy.sparse import csr_matrix
+
+from sklearn.utils.mocking import CheckingClassifier
+from sklearn.utils.testing import (assert_equal, assert_array_equal,
+                                   assert_false)
+from sklearn.utils.testing import SkipTest
+from sklearn.ensemble import (StackingTransformer, make_stack_layer)
+from sklearn.linear_model import (RidgeClassifier, LinearRegression)
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.svm import LinearSVC, LinearSVR, SVC
+from sklearn import datasets
+from sklearn.model_selection import (ParameterGrid, StratifiedKFold)
+
+iris = datasets.load_iris()
+X, y = iris.data[:, 1:3], iris.target
+
+RANDOM_SEED = 8939
+
+META_ESTIMATOR_PARAMS = {'cv': [2, StratifiedKFold()],
+                         'n_jobs': [1, 2]}
+META_ESTIMATOR_FIT_PARAMS = [{}, {"sample_weight": np.ones(y.shape)}]
+
+
+def _check_estimator(estimator, **fit_params):
+    # checks that we can fit_transform to the data
+    Xt = estimator.fit_transform(X, y, **fit_params)
+
+    # checks that we get a column vector
+    assert_equal(Xt.ndim, 2)
```

Comment:
why not check the shape exactly?

Reply:
after it borked on CI, I remembered why: this checks all estimators, so the number of columns may vary. We generally don't care about the amount of columns, just that it's a column vector, as different methods may yield different amounts of features